docs/devel/api-conventions.md

```diff
@@ -117,9 +121,15 @@ Kinds are grouped into three categories:
 
    Given their limited scope, they have the same set of limited common metadata as lists.
 
-   The "size" action may accept a simple resource that has only a single field as input (the number of things). The "status" kind is returned when errors occur and is not persisted in the system.
+   For instance, the "Status" kind is returned when errors occur and is not persisted in the system.
 
-   Examples: Binding, Status
+   Many simple resources are "subresources", which are rooted at API paths of specific resources. When resources wish to expose alternative actions or views that are closely coupled to a single resource, they should do so using new sub-resources. Common subresources include:
+
+   * `/binding`: Used to bind a resource representing a user request (e.g., Pod, PersistentVolumeClaim) to a cluster infrastructure resource (e.g., Node, PersistentVolume).
+   * `/status`: Used to write just the status portion of a resource. For example, the `/pods` endpoint only allows updates to `metadata` and `spec`, since those reflect end-user intent. An automated process should be able to modify status for users to see by sending an updated Pod kind to the server to the "/pods/&lt;name&gt;/status" endpoint - the alternate endpoint allows different rules to be applied to the update, and access to be appropriately restricted.
+   * `/scale`: Used to read and write the count of a resource in a manner that is independent of the specific resource schema.
+
```

Comment:
I'll add that in a later pass. I hope to find time to hash out initializers in a couple weeks.